target_pol/docs/style.css

```diff
@@ -0,0 +1,58 @@
+body {
```

Comment:
We have to move this file somewhere else: The idea is that `target_pol` is a directory for generated files which can always be safely deleted. The logic should be that the `pol doc` command creates this css file in the `target_pol` directory instead of assuming that it is already there.

Reply:
1. Generate the CSS if not there
2. Save it in a subfolder
3. Somewhere else in target_pol